### DISCUSSION TITLE
bromite relies on curbengh.github.io?

### DISCUSSION THREAD

**Discussion by Tanzin01** [bromite/bromite#1740] [Q&A] [answered]
In adblock I saw bromite's official link. However, it seems bromite connect to curbengh.github.io 

![Screenshot_20220128-175454](https://user-images.githubusercontent.com/41266558/151549111-d18f74a6-b839-45a0-98c7-26a53d514967.png)

**Comment by csagan5** [comment, marked as answer]
All domains under `github.io` use the same IP addresses; your app or OS cached the IP for that domain and incorrectly shows it as the reverse DNS lookup.
